test/libsolidity/SemanticTest.cpp

```diff
@@ -47,6 +53,25 @@ SemanticTest::SemanticTest(string const& _filename, langutil::EVMVersion _evmVer
 	m_lineOffset(m_reader.lineNumber()),
 	m_enforceViaYul(enforceViaYul)
 {
+	using namespace std::placeholders;
+	// clang-format off
```

Comment:
Are we using this in our code base? If not, I am not sure if we should leave these?

Reply:
removed.